library/std/src/net/ip.rs

```diff
@@ -122,16 +114,81 @@ pub struct Ipv6Addr {
     inner: c::in6_addr,
 }
 
-#[allow(missing_docs)]
+/// Scope of an IPv6 address as defined in [section 2 of IETF RFC 7346]
+///
+/// # Stability guarantees
+///
+/// This enum [may by subject to changes][changes] in the
+/// future, as new RFCs are published.
```

Comment:
Maybe this should clarify 'IETF RFCs'? At first I thought you meant Rust RFCs and was like well of course :laughing:

Reply:
lol good point